### ISSUE TITLE
Update COL project metadata

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#278]
Before the next release we need to:

 1. switch to the new metadata model by doing a prod deploy
 2. update the COL project metadata
 3. test a release

The updated COL metadata should include all team members as `contributors` with their role given in `note`.
We should also populate ISSN and discuss if/how to use publisher and contact. 

See also https://github.com/CatalogueOfLife/general/issues/88 for structured citations.

**Comment by mdoering** [comment]
@chantalhuijbers @yroskov do you have the list of team members to add to contributors?
We could actually already prepare a YAML file that we can then upload once we have deployed to prod.

**Comment by chantalhuijbers** [comment]
@mdoering Yes, this is the list that was shared in the Global Team meeting, it includes roles as I think it would be good to indicate who is in Global Team, the working groups, Board of Directors, Secretariat etc. https://docs.google.com/spreadsheets/d/1TkXR1A5z6dyuZoaF775Vb5V6UAR-Pc_iJvgS3N7WH58/edit?usp=sharing
